l1-contracts/test/foundry/unit/concrete/PriorityQueue/OnEmptyQueue.sol

```diff
@@ -0,0 +1,22 @@
+pragma solidity 0.8.20;
+
+import {PriorityQueueSharedTest} from "./_PriorityQueue_Shared.t.sol";
+
+contract OnEmptyQueueTest is PriorityQueueSharedTest {
+    function test_gets() public {
+        assertEq(0, priorityQueue.getSize());
+        assertEq(0, priorityQueue.getFirstUnprocessedPriorityTx());
+        assertEq(0, priorityQueue.getTotalPriorityTxs());
+        assertTrue(priorityQueue.isEmpty());
```

Comment:
```suggestion
        assertEq(0, priorityQueue.getPriorityQueueSize());
        assertTrue(priorityQueue.isEmpty());
```

Reply:
but priorityQueue has only 'getSize' method - that I check 3 lines above, and it doesn't have getPriorityQueueSize.
or am I missing something?

Reply:
You are right, ignore my comment above.